resources/ec2/v1alpha1/elastic-ip.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
this filename should be `resources/ec2/v1alpha1/elasticipaddress.yaml` to match the resource name :)